crates/cargo-lambda-watch/src/trigger_router.rs

```diff
@@ -243,3 +230,64 @@ async fn schedule_invocation(
 
     Ok(resp)
 }
+
+fn extract_path_parameters(path: &str) -> (String, String) {
+    let mut comp = path.split('/');
+
+    comp.next();
+    if let (Some(prefix), Some(fun_name)) = (comp.next(), comp.next()) {
+        if prefix == LAMBDA_URL_PREFIX {
+            let l = format!("/{prefix}/{fun_name}");
+            if path.starts_with(&l) {
```

Comment:
under what circumstances would this conditional fail?  it seems like it'll always be true once you've gotten to this point